recipes/ctat-splicing/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ctat-splicing" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TrinityCTAT/CTAT-SPLICING/releases/download/CTAT-SPLICING-v{{ version }}/CTAT-SPLICING.v{{ version }}.FULL.tar.gz
+  sha256: ea76acc5acead0fa24a574c0a1f8073668d8c9857346dfc7977ad10dd7f7fe8b
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('ctat-splicing', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf =2.69
+    - bzip2
```

Comment:
Only build **tools** should be listed here.
All dev libraries should be in `host`

Reply:
Thanks I'll see if that fixes some things after this test commit